docs/_source/guides/llms/practical_guides/create_dataset.md

```diff
@@ -381,8 +394,19 @@ record = rg.FeedbackRecord(
 
 To import the dataset to your Argilla instance with the `FeedbackDataset.push_to_argilla()` method. Once pushed, you will be able to see your dataset in the UI.
 
+::::{tab-set}
+
+:::{tab-item} Argilla 1.14.0 or higher
+```python
+remote_dataset = dataset.push_to_argilla(name="my-dataset", workspace="my-workspace")
+```
```

Comment:
```suggestion
```python
# push and save the remote dataset as a variable
# recommended if you are making updates later on (e.g. adding more records)
remote_dataset = dataset.push_to_argilla(name="my-dataset", workspace="my-workspace")

# or: push directly without saving the variable if you don't need to make changes
dataset.push_to_argilla(name="my-dataset", workspace="my-workspace")
```
```